locales/en/app.ftl

```diff
@@ -11,6 +11,7 @@
 -brand-HIBP = Have I Been Pwned
 -brand-fxa = Firefox Account
 -brand-Chrome = Chrome
+-brand-team-email = breach-alerts@mozilla.com
```

Comment:
Where is this used?

Reply:
More important. There's no reason to make this localizable at all

Reply:
It's used in `error-invalid-unsub-blurb`.